polaris-react/src/utilities/features/context.ts

```diff
@@ -2,6 +2,8 @@ import {createContext} from 'react';
 
 import type {FeaturesConfig} from './types';
 
+export const summerEditions2023ClassName = 'Polaris-summer-editions-2023';
```

Comment:
Could we pull this value from the `@shopify/polaris-tokens` package maybe? At least until we consolidate tokens.

Reply:
Was thinking about this as well. It's not a priority and kind of weird for tokens to be supplying a class name so I'm going think about it a bit more and follow up with that next week